apps/accounts/templates/registration/password_reset_email.txt

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}{% autoescape off %}Hello from {{ site_name }}!
+
+You're receiving this e-mail because you or someone else has requested a password for your user account at {{ site_domain }}.
```

Comment:
@lkbhitesh07 value of `site_domain` is not being populated in the screenshot you shared above. Can you please fix this.

Reply:
Done
![Screenshot from 2021-01-11 18-39-58](https://user-images.githubusercontent.com/44300735/104186799-f29bcd80-543c-11eb-9f83-7552bbd61d81.png)